pandas/core/tools/datetimes.py

```diff
@@ -359,7 +359,9 @@ def _convert_listlike(arg, box, format, name=None, tz=tz):
                     return DatetimeIndex(arg, tz=tz, name=name)
                 except ValueError:
                     pass
-
+            from pandas import Series
```

Comment:
this should also work for a DTI. 

use ABCSeries rather than importing.

I am not sure this is the best location for this.